Makefile

```diff
@@ -26,6 +26,7 @@ $(SRC_DIR)/%.ttf: $(SRC_DIR)/%.ufo $(SRC_DIR)/%.ufo/*.plist \
                   $(SRC_DIR)/%.ufo/glyphs*/contents.plist \
                   $(SRC_DIR)/%.ufo/data/com.github.fonttools.ttx/*.ttx \
                   $(SRC_DIR)/%.ufo/data/com.daltonmaag.vttLib.plist
+	ufonormalizer -m $<
```

Comment:
Fine for the moment.  As noted elsewhere, a Git-hook might be better;  but that would require either:

a. fixing `ufonormalize` upstream to return the current exit values
b. wrapping `ufonormalize` in the mean-time.

Reply:
I still find a bit weird that a target which is meant to build artifacts from the sources, is changing the sources in-place.
Imagine you run gcc on a .c source file and it changes it to use tabs instead of spaces.
Besides that it doesn't change the output, this also means that when running the target the second time, make that looks at timestamps will think the source ufo changed (because it did, wheel the formatting at least).